native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -332,8 +335,9 @@ static apr_status_t create_worker(proxy_server_conf *conf, proxy_balancer *balan
                 }
                 return APR_SUCCESS; /* Done Already existing */
             } else {
-                ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
-                             "Created: can't reuse worker as it for %s cleaning...", url);
+                //MODCLUSTER-448
+                ap_log_error(APLOG_MARK, APLOG_ERR, 0, server, SAMEURL, url);
```

Comment:
This does not cause the crash, but it is the first sign of misconfiguration; hence error message.